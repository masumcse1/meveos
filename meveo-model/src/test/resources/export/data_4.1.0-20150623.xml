<meveoExport version="4.1.0">
    <exportTemplate name="CustomerAccount" entityToExport="org.meveo.model.payments.CustomerAccount" canDeleteAfterExport="false">
        <parameters>
            <entry>
                <string>provider</string>
                <string>provider</string>
            </entry>
        </parameters>
        <classesToExportAsFull />
        <classesToExportAsId />
        <classesToIgnoreFKNotFound />
        <groupedTemplates />
    </exportTemplate>
    <data>
        <org.meveo.model.payments.CustomerAccount class="org.meveo.model.payments.CustomerAccount">
            <id>2068</id>
            <version>0</version>
            <provider id="2" code="DEMO" />
            <paymentMethod>WIRETRANSFER</paymentMethod>
            <creditCategory>NEWCUSTOMER</creditCategory>
            <dateDunningLevel class="sql-timestamp">2015-06-03 08:54:18.482</dateDunningLevel>
            <password>iHUgcKTU</password>
        </org.meveo.model.payments.CustomerAccount>
        <org.meveo.model.payments.CustomerAccount class="org.meveo.model.payments.CustomerAccount">
            <id>2069</id>
            <version>0</version>
            <provider id="2" code="DEMO" />
            <paymentMethod>WIRETRANSFER</paymentMethod>
            <creditCategory code="NEWCUSTOMER" provider="DEMO" />
            <dateDunningLevel class="sql-timestamp">2015-06-03 08:54:18.482</dateDunningLevel>
            <password>wVFcSlsw</password>
        </org.meveo.model.payments.CustomerAccount>
        <org.meveo.model.payments.DunningPlan>
            <id>1</id>
            <version>0</version>
            <provider id="1" code="DEMO" />
            <disabled>false</disabled>
            <auditable>
                <created class="sql-timestamp">2015-06-23 15:39:44.427</created>
            </auditable>
            <code>Dunning plan</code>
            <description>plan </description>
            <creditCategory>NEWCUSTOMER</creditCategory>
            <paymentMethod>CHECK</paymentMethod>
            <transitions />
            <actions />
            <status>ACTIVE</status>
        </org.meveo.model.payments.DunningPlan>
        <org.meveo.model.payments.DunningPlan>
            <id>1</id>
            <version>0</version>
            <provider id="1" code="DEMO" />
            <disabled>false</disabled>
            <auditable>
                <created class="sql-timestamp">2015-06-23 15:39:44.427</created>
            </auditable>
            <code>Dunning plan</code>
            <description>plan </description>
            <creditCategory code="NEWCUSTOMER" provider="DEMO" />
            <paymentMethod>CHECK</paymentMethod>
            <transitions />
            <actions />
            <status>ACTIVE</status>
        </org.meveo.model.payments.DunningPlan>
        <org.meveo.model.jobs.TimerEntity>
            <id>1</id>
            <version>0</version>
            <provider id="1" code="DEMO" />
            <name>A - Import Customers</name>
            <jobName>ImportCustomersJob</jobName>
            <timerInfo>
                <active>false</active>
                <jobName>ImportCustomersJob</jobName>
                <userId>1</userId>
            </timerInfo>
            <year>*</year>
            <month>*</month>
            <dayOfMonth>*</dayOfMonth>
            <dayOfWeek>*</dayOfWeek>
            <hour>*</hour>
            <minute>0</minute>
            <second>0</second>
            <jobCategoryEnum>ACCOUNT_RECEIVABLES</jobCategoryEnum>
        </org.meveo.model.jobs.TimerEntity>
        <org.meveo.model.jobs.TimerEntity xsId="1">
            <id>1</id>
            <version>0</version>
            <provider xsId="2" code="DEMO" id="1" />
            <disabled>false</disabled>
            <auditable xsId="4">
                <created xsId="5" class="sql-timestamp">2015-06-22 21:14:12.109</created>
            </auditable>
            <code>code_1</code>
            <description />
            <year>*</year>
            <month>*</month>
            <dayOfMonth>*</dayOfMonth>
            <dayOfWeek>*</dayOfWeek>
            <hour>*</hour>
            <minute>0</minute>
            <second>0</second>
        </org.meveo.model.jobs.TimerEntity>
        <org.meveo.model.crm.CustomFieldInstance>
            <id>539</id>
            <version>0</version>
            <provider code="DEMO" />
            <disabled>false</disabled>
            <auditable>
                <created class="sql-timestamp">2015-06-23 18:15:05.042</created>
            </auditable>
            <code>PDFInvoiceGenerationJob_nbRuns</code>
            <timerEntity id="8" name="8_PDF" provider="DEMO" />
            <longValue>1</longValue>
        </org.meveo.model.crm.CustomFieldInstance>
        <org.meveo.model.crm.CustomFieldInstance>
            <id>539</id>
            <version>0</version>
            <provider code="DEMO" />
            <disabled>false</disabled>
            <auditable>
                <created class="sql-timestamp">2015-06-23 18:15:05.042</created>
            </auditable>
            <code>PDFInvoiceGenerationJob_nbRuns</code>
            <timerEntity reference="../../org.meveo.model.crm.CustomFieldInstance[539]/timerEntity" />
            <longValue>1</longValue>
        </org.meveo.model.crm.CustomFieldInstance>
        <org.meveo.model.crm.CustomFieldInstance>
            <id>539</id>
            <version>0</version>
            <provider code="DEMO" />
            <disabled>false</disabled>
            <auditable>
                <created class="sql-timestamp">2015-06-23 18:15:05.042</created>
            </auditable>
            <code>PDFInvoiceGenerationJob_nbRuns</code>
            <jobInstance id="8" code="8_PDF" provider="DEMO" />
            <longValue>1</longValue>
            <valuePeriods/>
        </org.meveo.model.crm.CustomFieldInstance>
        <org.meveo.model.catalog.CounterTemplate>
            <id>17</id>
            <version>0</version>
            <provider id="1" code="OCB" />
            <disabled>false</disabled>
            <auditable>
                <created class="sql-timestamp">2014-12-24 10:40:07.186</created>
            </auditable>
            <code>FCA_COMPT_SUPPORT_GOLD</code>
            <description>FCA Compteur Support Gold</description>
            <calendar class="org.meveo.model.catalog.CalendarYearly" id="4" code="MONTHLY" provider="OCB" />
            <ceiling>7900.000000000000</ceiling>
            <unityDescription>Euros</unityDescription>
        </org.meveo.model.catalog.CounterTemplate>
        <org.meveo.model.catalog.CounterTemplate>
            <id>18</id>
            <version>0</version>
            <provider reference="../../org.meveo.model.catalog.CounterTemplate/provider" />
            <disabled>false</disabled>
            <auditable>
                <created class="sql-timestamp">2014-12-24 10:40:52.236</created>
            </auditable>
            <code>FCA_COMPT_SUPPORT_SILVER</code>
            <description>FCA Compteur Support Silver</description>
            <calendar class="org.meveo.model.catalog.CalendarYearly" reference="../../org.meveo.model.catalog.CounterTemplate/calendar" />
            <ceiling>7800.000000000000</ceiling>
            <unityDescription>Euros</unityDescription>
            <counterType>USAGE</counterType>
            <counterLevel>UA</counterLevel>
        </org.meveo.model.catalog.CounterTemplate>
    </data>
</meveoExport>