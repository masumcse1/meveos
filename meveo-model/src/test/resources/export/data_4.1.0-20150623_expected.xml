<meveoExport version="4.1.0-20150623">
    <exportTemplate name="CustomerAccount" entityToExport="org.meveo.model.payments.CustomerAccount" canDeleteAfterExport="false">
        <parameters>
            <entry>
                <string>provider</string>
                <string>provider</string>
            </entry>
        </parameters>
        <classesToExportAsFull />
        <classesToExportAsId />
        <classesToIgnoreFKNotFound />
        <groupedTemplates />
    </exportTemplate>
    <data>
        <org.meveo.model.payments.CustomerAccount class="org.meveo.model.payments.CustomerAccount">
            <id>2068</id>
            <version>0</version>
            <provider id="2" code="DEMO" />
            <paymentMethod>WIRETRANSFER</paymentMethod>
            <creditCategory code="NEWCUSTOMER" provider="DEMO" />
            <dateDunningLevel class="sql-timestamp">2015-06-03 08:54:18.482</dateDunningLevel>
            <password>iHUgcKTU</password>
        </org.meveo.model.payments.CustomerAccount>
        <org.meveo.model.payments.CustomerAccount class="org.meveo.model.payments.CustomerAccount">
            <id>2069</id>
            <version>0</version>
            <provider id="2" code="DEMO" />
            <paymentMethod>WIRETRANSFER</paymentMethod>
            <creditCategory code="NEWCUSTOMER" provider="DEMO" />
            <dateDunningLevel class="sql-timestamp">2015-06-03 08:54:18.482</dateDunningLevel>
            <password>wVFcSlsw</password>
        </org.meveo.model.payments.CustomerAccount>
        <org.meveo.model.payments.DunningPlan>
            <id>1</id>
            <version>0</version>
            <provider id="1" code="DEMO" />
            <disabled>false</disabled>
            <auditable>
                <created class="sql-timestamp">2015-06-23 15:39:44.427</created>
            </auditable>
            <code>Dunning plan</code>
            <description>plan </description>
            <creditCategory code="NEWCUSTOMER" provider="DEMO" />
            <paymentMethod>CHECK</paymentMethod>
            <transitions />
            <actions />
            <status>ACTIVE</status>
        </org.meveo.model.payments.DunningPlan>
        <org.meveo.model.payments.DunningPlan>
            <id>1</id>
            <version>0</version>
            <provider id="1" code="DEMO" />
            <disabled>false</disabled>
            <auditable>
                <created class="sql-timestamp">2015-06-23 15:39:44.427</created>
            </auditable>
            <code>Dunning plan</code>
            <description>plan </description>
            <creditCategory code="NEWCUSTOMER" provider="DEMO" />
            <paymentMethod>CHECK</paymentMethod>
            <transitions />
            <actions />
            <status>ACTIVE</status>
        </org.meveo.model.payments.DunningPlan>
        <org.meveo.model.jobs.TimerEntity>
            <id>1</id>
            <version>0</version>
            <provider id="1" code="DEMO" />
            <auditable>
                <created class="sql-timestamp">2015-06-23 18:13:23.166</created>
            </auditable>
            <code>code_1</code>
            <year>*</year>
            <month>*</month>
            <dayOfMonth>*</dayOfMonth>
            <dayOfWeek>*</dayOfWeek>
            <hour>*</hour>
            <minute>0</minute>
            <second>0</second>
            <jobInstances>
                <org.meveo.model.jobs.JobInstance>
                    <id>1</id>
                    <version>0</version>
                    <provider code="DEMO" />
                    <auditable>
                        <created class="sql-timestamp">2015-06-23 18:13:23.166</created>
                    </auditable>
                    <userId>1</userId>
                    <code>A - Import Customers</code>
                    <jobTemplate>ImportCustomersJob</jobTemplate>
                    <jobCategoryEnum>ACCOUNT_RECEIVABLES</jobCategoryEnum>
                    <timerEntity reference="../../.." />
                </org.meveo.model.jobs.JobInstance>
            </jobInstances>
        </org.meveo.model.jobs.TimerEntity>
        <org.meveo.model.jobs.TimerEntity xsId="1">
            <id>1</id>
            <version>0</version>
            <provider xsId="2" code="DEMO" id="1" />
            <disabled>false</disabled>
            <auditable xsId="4">
                <created xsId="5" class="sql-timestamp">2015-06-22 21:14:12.109</created>
            </auditable>
            <code>code_1</code>
            <description />
            <year>*</year>
            <month>*</month>
            <dayOfMonth>*</dayOfMonth>
            <dayOfWeek>*</dayOfWeek>
            <hour>*</hour>
            <minute>0</minute>
            <second>0</second>
        </org.meveo.model.jobs.TimerEntity>
        <org.meveo.model.crm.CustomFieldInstance>
            <id>539</id>
            <version>0</version>
            <provider code="DEMO" />
            <disabled>false</disabled>
            <auditable>
                <created class="sql-timestamp">2015-06-23 18:15:05.042</created>
            </auditable>
            <code>PDFInvoiceGenerationJob_nbRuns</code>
            <jobInstance id="8" code="8_PDF" provider="DEMO" />
            <longValue>1</longValue>
        </org.meveo.model.crm.CustomFieldInstance>
        <org.meveo.model.crm.CustomFieldInstance>
            <id>539</id>
            <version>0</version>
            <provider code="DEMO" />
            <disabled>false</disabled>
            <auditable>
                <created class="sql-timestamp">2015-06-23 18:15:05.042</created>
            </auditable>
            <code>PDFInvoiceGenerationJob_nbRuns</code>
            <jobInstance reference="../../org.meveo.model.crm.CustomFieldInstance[539]/jobInstance" />
            <longValue>1</longValue>
        </org.meveo.model.crm.CustomFieldInstance>
        <org.meveo.model.crm.CustomFieldInstance>
            <id>539</id>
            <version>0</version>
            <provider code="DEMO" />
            <disabled>false</disabled>
            <auditable>
                <created class="sql-timestamp">2015-06-23 18:15:05.042</created>
            </auditable>
            <code>PDFInvoiceGenerationJob_nbRuns</code>
            <jobInstance id="8" code="8_PDF" provider="DEMO" />
            <longValue>1</longValue>
            <valuePeriods/>
        </org.meveo.model.crm.CustomFieldInstance>
        <org.meveo.model.catalog.CounterTemplate>
            <id>17</id>
            <version>0</version>
            <provider id="1" code="OCB" />
            <disabled>false</disabled>
            <auditable>
                <created class="sql-timestamp">2014-12-24 10:40:07.186</created>
            </auditable>
            <code>FCA_COMPT_SUPPORT_GOLD</code>
            <description>FCA Compteur Support Gold</description>
            <calendar class="org.meveo.model.catalog.CalendarYearly" id="4" code="MONTHLY" provider="OCB" />
            <ceiling>7900.000000000000</ceiling>
            <unityDescription>Euros</unityDescription>
        <counterType>USAGE</counterType>
            <counterLevel>UA</counterLevel>
        </org.meveo.model.catalog.CounterTemplate>
        <org.meveo.model.catalog.CounterTemplate>
            <id>18</id>
            <version>0</version>
            <provider reference="../../org.meveo.model.catalog.CounterTemplate/provider" />
            <disabled>false</disabled>
            <auditable>
                <created class="sql-timestamp">2014-12-24 10:40:52.236</created>
            </auditable>
            <code>FCA_COMPT_SUPPORT_SILVER</code>
            <description>FCA Compteur Support Silver</description>
            <calendar class="org.meveo.model.catalog.CalendarYearly" reference="../../org.meveo.model.catalog.CounterTemplate/calendar" />
            <ceiling>7800.000000000000</ceiling>
            <unityDescription>Euros</unityDescription>
            <counterType>USAGE</counterType>
            <counterLevel>UA</counterLevel>
        </org.meveo.model.catalog.CounterTemplate>
    </data>
</meveoExport>
